prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -1037,3 +1037,39 @@ periodics:
         requests:
           memory: 200Mi
           cpu: 80m
+
+- name: kyma-gke-compass-integration-periodic
+  cron: "00 00 * * *" # "At 00:00 every day"
+  skip_report: false
```

Comment:
this is not a valid property for a periodic job

Reply:
`skip_report` removed